package.json

```diff
@@ -9,18 +9,29 @@
     "prettier:check": "prettier --check .",
     "prettier:write": "prettier --write .",
     "start": "react-native start",
-    "test": "jest"
+    "test": "NODE_OPTIONS='--no-deprecation' jest --watch --verbose",
```

Comment:
Why are we disabling deprecation warnings?

Reply:
I suppose we don't have to. Theres a couple items that react native is using which get pulled up by the depreciation warning, but from what i could find i don't think we can really update that dependency.

`(node:34618) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)`